src/Hamiltonians/BoseHubbardMom1D.jl

```diff
@@ -123,7 +123,7 @@ function get_offdiagonal(ham::BoseHubbardMom1D, add::ADDRESS, chosen) where ADDR
     onproduct *= occ + 1
     onr[ppq] = occ + 1
 
-    return ADDRESS(onr), ham.u/(2*ham.m)*sqrt(onproduct)
+    return ADDRESS(onr.data), ham.u/(2*ham.m)*sqrt(onproduct)
```

Comment:
Is it safe to use `onr.data`? It seems like an implementation detail that could break with future versions of StaticArrays.